src/app/projects/jira-key/jira-key.component.html

```diff
@@ -0,0 +1,32 @@
+<p-toast position="top-center"></p-toast>
+<div class="ui-g" id="papaya-box">
+  <div class="ui-g-8 ui-g-nopad">
+    <h5>Konfiguracja Jiry 1</h5>
+
+    <div class="ui-g-12">
+      <p class="first">Klucz publiczny: {{jiraConfig.key}}</p>
+    </div>
+
+    <div class="ui-g-12">
+      <p>Intrukcja:
+      <a href="https://developer.atlassian.com/server/jira/platform/oauth/">
```

Comment:
Tu by można dać konkretnie to https://developer.atlassian.com/server/jira/platform/oauth/#step-1--configure-jira

(Pewnie by wypadało Ładną Instrukcję zrobić, ale _no nie_)